sdk/include/opentelemetry/sdk/logs/exporter.h

```diff
@@ -47,6 +47,12 @@ class LogRecordExporter
   virtual sdk::common::ExportResult Export(
       const nostd::span<std::unique_ptr<Recordable>> &records) noexcept = 0;
 
+  /**
+   * Force flush the log records pushed into this log exporter.
+   */
+  virtual bool ForceFlush(
+      std::chrono::microseconds timeout = (std::chrono::microseconds::max)()) noexcept = 0;
```

Comment:
Should we have the default implementation for this method returning `true`, this will ensure that external exporters needn't add this method if they don't need. Same for span exporter.

Reply:
Good point, I will add it tomorrow.